.evergreen/run-mongosh-integration-tests.sh

```diff
@@ -4,7 +4,14 @@ set -o xtrace   # Write all commands first to stderr
 
 source "${PROJECT_DIRECTORY}/.evergreen/init-nvm.sh"
 
+npm cache clear --force || true
+npm i -g npm@8.x || true
+
 npm pack | tee npm-pack.log
+
+npm cache clear --force || true
+npm i -g npm@8.x || true
```

Comment:
Not entirely ensure *why* these are necessary but we're doing something similar in mongosh already. I can narrow down which of these statements are needed, if that's desirable.

Reply:
If I recall the cache clearing was specific to issues around windows and npm.

Reply:
Yeah, this was only causing trouble on Windows :+1: